packages/core/src/lib/time-tracking/time-slot/utils.ts

```
@@ -59,17 +59,13 @@
+	start: string | Date;
+	end: string | Date;
 } {
 	let startMinute = moment(start).utc().get('minute');
 	startMinute = startMinute - (startMinute % 10);
 
-	let startDate: any = moment(start)
-		.utc()
-		.set('minute', startMinute)
-		.set('second', 0)
-		.set('millisecond', 0);
+	let startDate: any = moment(start).utc().set('minute', startMinute).set('second', 0).set('millisecond', 0);
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'startDate' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)